tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/Microsoft.DotNet.SourceBuild.Tasks.XPlat.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netcoreapp5.0</TargetFramework>
```

Comment:
Why not stick with lowest applicable?

Reply:
I tried `netstandard2.1`, `netcoreapp3.0`, and `netcoreapp3.1`, and got the same error:
```
CSC : error CS1705: Assembly 'NuGet.Packaging' with identity 'NuGet.Packaging, Version=5.7.0.7, Culture=neutral, PublicKeyToken=31bf3856ad364e35' uses 'System.Runtime, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' which has a higher version than referenced assembly 'System.Runtime' with identity 'System.Runtime, Version=4.2.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' [/home/chris/dotnet_source-build/tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/Microsoft.DotNet.SourceBuild.Tasks.XPlat.csproj]
BuildDummyPackages.cs(55,34): error CS7069: Reference to type 'FrameworkName' claims it is defined in 'System.Runtime', but it could not be found [/home/chris/dotnet_source-build/tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/Microsoft.DotNet.SourceBuild.Tasks.XPlat.csproj]
```

I also tried upgrading the MSBuild versions we use and got pretty similar errors:
```
BuildDummyPackages.cs(55,34): error CS7069: Reference to type 'Stream' claims it is defined in 'System.Runtime', but it could not be found [/home/chris/dotnet_source-build/tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/Microsoft.DotNet.SourceBuild.Tasks.XPlat.csproj]
BuildDummyPackages.cs(55,34): error CS7069: Reference to type 'FrameworkName' claims it is defined in 'System.Runtime', but it could not be found [/home/chris/dotnet_source-build/tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/Microsoft.DotNet.SourceBuild.Tasks.XPlat.csproj]
```
Any idea on those?